examples/plot_simulate_evoked.py

```diff
@@ -42,12 +43,19 @@
 import matplotlib.pyplot as plt
 fig, axes = plt.subplots(2, 1, sharex=True, figsize=(6, 6))
 for dpl in dpls:
-    dpl.plot(ax=axes[0], layer='agg')
+    dpl.plot(ax=axes[0], layer='agg', show=False)
 net.plot_input(ax=axes[1])
 
 ###############################################################################
-# Finally, we can also plot the spikes.
-net.plot_spikes()
+# Also, we can plot the spikes and write them to txt files.
+# Note that we can use formatting syntax to specify the filename pattern
+# with which each trial will be written. To read spikes back in, we can use
+# wildcard expressions.
+with tempfile.TemporaryDirectory() as tmp_dir_name:
+    print(tmp_dir_name)
+    net.spikes.write(op.join(tmp_dir_name, 'spk_%d.txt'))
+    spikes = read_spikes(op.join(tmp_dir_name, 'spk_*.txt'))
+spikes.plot()
```

Comment:
I get that it's good to demo the use of the `write` method and `read_spikes` function. But I wonder if it confuses the new user (likely to look at this example first). The writing to and reading from disk are not _needed_ for plotting. Maybe both? First `net.spikes.plot()`, then the I/O-bit?